### PR TITLE
Topic IgxList panning templates

### PR DESCRIPTION
**Pull request by rmkrmk**
Left and right panning template for IgxList
IgniteUI/igniteui-angular#728

### CODE REVIEW

**Comment by tachojelev** [comment]
Typo:

On line **221**, it should be:
 // contacts.**component**.ts
instead of:
 // contacts.**comoponent**.ts

File: en/components/list.md [hunk 1]
```diff
@@ -448,6 +591,15 @@ The following properties are available on the **igx-list-item** component:
 
 <div class="divider"></div>
 
+The following properties are available on the `IListItemPanningEventArgs` type:
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `item` | IgxListItemComponent | A reference to the **igx-list-item** being panned. |
+| `direction` | IgxListPanState | Specifies the direction at which the **igx-list-item** has been panned. |
+| `keepItem` | Boolean | If `true` the list item will be kept in the **IgxList** after it has been panned. If `false` (by default) the list item will be removed form the **IgxList** after it has been panned. |
```
[the remark this answers is not in the document]
Comment (rmkrmk) [reply en/components/list.md]: Fixed

File: jp/components/list.md [hunk 2]
```diff
@@ -330,6 +330,152 @@ toggleFavorite(item: IgxListItem) {
...
+      <igx-icon name="star" [color]="contact.isFavorite ? 'orange' : 'lightgray'"
+        (click)="toggleFavorite(item)"></igx-icon>
+    </div>
+  </igx-list-item>
+</igx-list>
+
+<igx-toast #toast></igx-toast>
+```
+
+The above example is using some CSS styles which may be found here:
```
Comment (ig-ealbert) [inline jp/components/list.md]: @yoshimis Let's translate this line?

File: jp/components/list.md [hunk 3]
```diff
@@ -330,6 +330,152 @@ toggleFavorite(item: IgxListItem) {
...
+> [!NOTE]
+> リスト項目のパンニング時にパンニング イベントが発生するために達する必要のあるしきい値があります。**IgxList's** `panEndTriggeringThreshold` プロパティを使用するしきい値を変更できます。このプロパティのデフォルトは0.5 でリスト項目幅の 50% を意味します。
+
+次にリスト項目をパンニングします。
+
+<div class="sample-container loading" style="height: 477px">
+<iframe id="list-sample-7-final-iframe" src='{environment:demosBaseUrl}/list-sample-7' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="list-sample-7-final-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
```
Comment (ig-ealbert) [inline jp/components/list.md]: Translate the button text.

File: jp/components/list.md [hunk 4]
```diff
@@ -449,6 +596,15 @@ toggleFavorite(item: IgxListItem) {
 
 <div class="divider"></div>
 
+以下のプロパティは `IListItemPanningEventArgs` 型で使用できます。
+| 名前 | 型 | 説明 |
+| :--- | :--- | :--- |
+| `item` | IgxListItemComponent | パンニングされている **igx-list-item** の参照。 |
+| `direction` | IgxListPanState | **Igx-list-item** がパンニングされる方向を指定します。 |
```
Comment (ig-ealbert) [inline jp/components/list.md]: I think this should be **igx-list-item** (lowercase).

File: jp/components/list.md (hunk 2 above)
Comment (ig-ealbert) [inline jp/components/list.md]: @yoshimis Let's translate this line?

**Pull request merged**